### PR TITLE
Improved README

### PR DESCRIPTION
**Pull request by apurvis**
@kamui this includes all the changes from https://github.com/kamui/retriable/pull/44 but also with an update to the default behavior section... kept it separate in case you weren't a fan of these changes.

I took out the non randomized intervals calculation, because that has nothing to do with the default behavior, and i rounded to the nearest millisecond, because i'm not sure those extra precision points were doing anything except making it a lot harder to read the table.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,21 +35,29 @@ gem 'retriable', '~> 3.1'
...
-| 5        |    2.53125     |  [1.265625, 3.796875]           |
-| 6        |    3.796875    |  [1.8984375,  5.6953125]        |
-| 7        |    5.6953125   |  [2.84765625,  8.54296875]      |
-| 8        |   8.54296875   |  [4.271484375, 12.814453125]    |
-| 9        |  12.814453125  |  [6.4072265625, 19.2216796875]  |
-| 10       | 19.2216796875  |  stop                           |
+Code in a `Retriable.retriable` block will be retried if an exception is raised.
+
+
+### Defaults
```
Comment (kamui) [inline README.md]: I like this new defaults, but I actually do like having the interval there too.

Comment (apurvis) [inline README.md]: the interval is still there - it's the non randomized value that has been deleted.  i'll close the other PR though and just fix rubinius here.

Comment (apurvis) [inline README.md]: how about if there's just a middle column for "average"? so then you have min/avg/max for the defaults.

Comment (kamui) [inline README.md]: Sounds good to me 👍

**Comment by kamui** [comment]
Thanks for the PR!

**Pull request merged**